backend/follows/api/views.py

```diff
@@ -78,3 +78,15 @@ def get(self, request):
             return Response(response_load, status=status.HTTP_200_OK)
         except ObjectDoesNotExist:
             return Response(response_load, status=status.HTTP_400_BAD_REQUEST)
+
+class deleteFollowAPI(APIView):
+    def delete(self, request):
+        following_id = request.data["following_id"]
+        follower_id = request.data["follower_id"]
+
+        try:
+            followObj = Follow.objects.filter(follower_id=follower_id, following_id=following_id)
+            followObj.delete()
+            return Response(status=status.HTTP_200_OK)
+        except ObjectDoesNotExist:
+            return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
1. Related to No.3 above. Your implementation has extra step, it will take extra time on filtering/searching the right data.
2. No validation, another user can delete the follows data

Todo:
1. delete the record by using id/pk
2. add extra validation where the follower_id === current login user_id because right now, any user can delete any data by manipulating your payload